pkg/memo/memo_test.go

```diff
@@ -0,0 +1,264 @@
+package memo_test
```

Comment:
On the memo test, why not having a mapping bytes => expected struct directly in the TDT?

Reply:
Do you mean replacing the existing `headBytes + dataBytes` ==> `expectedMemo` as one single `dataBytes` ==> `expectedMemo`, or visualizing the bytes as hex string like below:

```
dataHex: `0000000000000000000000005ec58c3c001fa30726accfba71bdd90ce4f56b08` +
	`00000000000000000000000000000000000000000000000000000000000000a0` +
	`00000000000000000000000000000000000000000000000000000000000000e0` +
	`0000000000000000000000005ec58c3c001fa30726accfba71bdd90ce4f56b08` +
	`0000000000000000000000000000000000000000000000000000000000000120` +
	`0000000000000000000000000000000000000000000000000000000000000001` +
	`7686572655f735f736f6d655f62797465735f6669656c6400000000000000000` +
	`0000000000000000000000000000000000000000000000000000000000000000` +
	`16746869735f69735f615f737472696e675f6669656c64000000000000000000` +
	`0000000000000000000000000000000000000000000000000000000000000000` +
	`17686572655f735f736f6d655f62797465735f6669656c640000000000000000`,
```